src/rpp/rpp/observables/specific_observable.hpp

```diff
@@ -104,7 +104,10 @@ class specific_observable : public interface_observable<Type, specific_observabl
     }
 
 private:
-    RPP_NO_UNIQUE_ADDRESS OnSubscribeFn m_state;
+    /**
+     * \brief The on_subscribe functor (type-erasure), which has the operator()(const auto& subscriber) overload function.
```

Comment:
There is no any type-erasure :) we use some specific type,not something like std::function